source/volume_of_fluid/handler.cc

```diff
@@ -166,9 +166,9 @@ namespace aspect
           neighbor_dof_indices (data.neighbor_dof_indices)
         {
           // clear the flag that indicates that we have valid data in any
-          // of the matrices
-          for (unsigned int i=0; i < face_contributions_mask.size(); ++i)
-            face_contributions_mask[i] = false;
+          // of the matrices:
+          for (auto &&flag : face_contributions_mask)
+            flag = false;
```

Comment:
This is correct, but it is difficult to understand why it is so (because `std::vector<bool>` has an accessor class that you get when dereferencing iterators). Perhaps it is not important, but out of curiosity, what was the warning you got?

Reply:
clang-tidy wants to modernize all for loop and turn them into range-based loops. Not sure how else to silence this warning. Maybe 
```
face_contributions_mask.assign(face_contributions_mask.size(), false);
```
?